changelog.d/20240627_181028_chris_mep_config_user_runtime.rst

```diff
@@ -0,0 +1,7 @@
+New Functionality
+^^^^^^^^^^^^^^^^^
+
+- MEP config templates now receive information about the runtime environment used to
+  submit tasks, such as Python environment and Globus Compute SDK version, via the
+  ``user_runtime`` variable. `See here <https://globus-compute.readthedocs.io/en/latest/reference/client.html#globus_compute_sdk.sdk.batch.UserRuntime>`_
+  for a complete list of the fields that are sent.
```

Comment:
In terms of HTML netizenry, the long-standing advice is to not link "click here" or "here" or "see here" (etc.), but to link a more descriptive set of text.  Suggest something like:
```rest
For the complete list of fields, please reference the `batch.UserRuntime documentation <...>`_.